xbmc/guilib/TextureBundle.cpp

```diff
@@ -18,15 +18,10 @@
  *
  */
 
-#include "system.h"
 #include "TextureBundle.h"
 
-CTextureBundle::CTextureBundle(void)
-{
-  m_useXBT = false;
-}
-
-CTextureBundle::~CTextureBundle(void)
+CTextureBundle::CTextureBundle()
+  : m_useXBT{false}
```

Comment:
Nice. Cares for narrowing.
Am 06.07.2016 9:48 vorm. schrieb "Pär Björklund" <notifications@github.com>:

> In xbmc/guilib/TextureBundle.cpp
> <https://github.com/xbmc/xbmc/pull/9960#discussion_r69687319>:
>
> >  #include "TextureBundle.h"
> >
> > -CTextureBundle::CTextureBundle(void)
> > -{
> > -  m_useXBT = false;
> > -}
> > -
> > -CTextureBundle::~CTextureBundle(void)
> > +CTextureBundle::CTextureBundle()
> > +  : m_useXBT{false}
>
> That is correct since c++11, also preferred since it breaks the build if
> you accidentally perform implicit conversion during init
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/9960/files/d43650defcef6bdc0f04d8af4a764d7219527313#r69687319>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABCfHT0f_WczupAjA_CWH-CQUb_zKbbkks5qS13ZgaJpZM4Izxgy>
> .
>